src/main/java/org/opentripplanner/graph_builder/module/osm/DefaultWayPropertySetSource.java

```diff
@@ -369,6 +369,8 @@ public void populateProperties(WayPropertySet props) {
         props.setCarSpeed("highway=service", 6.7f); // ~= 15 mph
         props.setCarSpeed("highway=track", 4.5f); // ~= 10 mph
         props.setCarSpeed("highway=road", 11.2f); // ~= 25 mph
+        props.setCarSpeed("surface=gravel", 4.5f); // ~= 10 mph
+        props.setCarSpeed("surface=dirt", 4.5f); // ~= 10 mph
```

Comment:
It does because the Micromobility speed is capped at the maximum allowed car speed. I was seeing some shortcuts along lots of gravel roads being taken when trying out some routes, so I added this.